test/functional/apps/management/_import_objects.js

```diff
@@ -48,13 +49,26 @@ export default function ({ getService, getPageObjects }) {
         );
         await PageObjects.savedObjects.checkImportSucceeded();
         await PageObjects.savedObjects.clickImportDone();
+        await PageObjects.common.sleep(5000);
 
         // get all the elements in the table, and index them by the 'title' visible text field
         const elements = keyBy(await PageObjects.savedObjects.getElementsInTable(), 'title');
         log.debug("check that 'Log Agents' is in table as a visualization");
         expect(elements['Log Agents'].objectType).to.eql('visualization');
 
-        await elements['logstash-*'].relationshipsElement.click();
+        // there's either a menu of actions
+        if (elements['logstash-*'].menuElement) {
+          await elements['logstash-*'].menuElement?.click();
+          // Wait for context menu to render
+          const menuPanel = await find.byCssSelector('.euiContextMenuPanel');
+          await (
+            await menuPanel.findByTestSubject('savedObjectsTableAction-relationships')
+          ).click();
+        } else {
+          // or the action elements are on the row without the menu
+          await elements['logstash-*'].relationshipsElement.click();
+        }
```

Comment:
I would like to refactor this whole block of functionality to be in the saved_objects_page so we could just call something like `await PageObjects.savedObjects.releationshipsClick('logstash-*');` and it would open the context menu if it had to.  Just not sure if it makes sense to put more work into it now or just get these tests passing reliably.